app/src/main/kotlin/org/eclipse/kuksa/companion/feature/sheet/view/BottomSheetView.kt

```diff
@@ -46,7 +45,7 @@ fun BottomSheetView(
     modifier: Modifier = Modifier,
     initialSheetValue: SheetValue = SheetValue.PartiallyExpanded,
     sheetContent: @Composable () -> Unit = {},
-    content: @Composable (PaddingValues) -> Unit,
```

Comment:
for the sideSheet the PaddingValues are unused (0.dp) for the BottomSheet the PaddingValues contain a "bottomPadding" with the height of the BottomSheet (unused in our RamsesView). Removed these PaddingValues so it does not look like they are forgotten in AdaptiveAppScreen.